### DISCUSSION TITLE
Cursor Hover seems to not work with new iPads and Apple Pencil Pro

### DISCUSSION THREAD

**Discussion by joelmayer87** [keiwando/toonsquid#582] [General]
I definitely have it activated in the Toonsquid Settings but i unfortunately see no hovering cursor. This might be normal though because the new Pencil was just released and maybe the apps needs some adjustments but i thought i'd just give a heads up.

**Comment by DinoAwakening** [comment]
I mean, it needs an update from each app’s developers. The new Apple Pencil Pro just came out, and the last big update we got on ToonSquid was almost 8 months ago. Unless Keiwan magically predicted that a new Apple Pencil was on the way, he has to either delay the 2.0, or not add any feature relate to the Apple Pencil until the next update.

**Reply by joelmayer87** [reply]
Sure, it wasn't a criticism of the developer, i was merely pointing it out since it would be understandable if he hasn't a new iPad (yet).

**Comment by keiwando** [comment]
Yes, that is currently expected. I will have to provide support for those features with an app update. None of that stuff works automatically. 
The hovering feature was already introduced with the previous generation of iPad Pros, but so far there have been barely any requests for me to add support for it, so it was relatively low on the priority list. Now with the Pencil Pro that also has the barrel roll and squeeze features and is supported by the new iPad Air as well, the priority has increased a bit. 
That being said, I'm currently still mainly focused on trying to finish up the 2.0 update and get it ready for beta testing. I'll probably look deeper into the new Apple Pencil Pro features afterwards.

**Reply by joelmayer87** [reply]
I just saw that in the Toonsquid settings there was a hover option :) Personally, i really like it since i can see the size of the pen etc. but again, this wasn't meant as a negative, just thought i'd mention it here.

2.0 is definitely a higher priority and other than the hover, Toonsquid works like a charm on an M4 with the new Pro Pencil :)

Can't wait for 2.0 and thanks for all your hard work! Great to see that Toonsquid seems to be so successful on the Appstore as well ^^

**Reply by CowlBlue** [reply]
Yes! Very excited for the 2.0 update!

**Reply by Cjjuntti** [reply]
So, we’re days away from seeing the new 2.0 update? Hell yeah! Let’s go Keiwan! You’re the freaking man!

**Reply by joelmayer87** [reply]
Woah where did he ever say we were "days" away from a 2.0 update :D In software development "soon" can mean many things from weeks to months. Give him a break lol.

**Reply by core999** [reply]
just writing a comment to say I also wouldn't mind seeing the hover and barrel squeeze features be looked into after the 2.0 as well :)

 I had the old 2017 ipad and seeing my cursor and getting to use new gestures with the new ipad is actually a nicer QoL update than I expected.
